### PR TITLE
animal-table PR

### PR DESCRIPTION
**Pull request by gilberthappi**
.CREATE database
.CREATE table

 * Write queries for the following quests:
>   
>   * Find all animals whose name ends in "mon".
>     Screenshot 2022-10-17 174142
>   * List the name of all animals born between 2016 and 2019.
>     Screenshot 2022-10-17 180232
>   * List the name of all animals that are neutered and have less than 3 escape attempts.
>     Screenshot 2022-10-17 180302
 * Find all animals that are neutered.
>     Screenshot 2022-10-17 180438
>   * Find all animals not named Gabumon.
>     Screenshot 2022-10-17 180530
>   * Find all animals with a weight between 10.4kg and 17.3kg
>     Screenshot 2022-10-17 180609

>   * List the date of birth of all animals named either "Agumon" or "Pikachu".
>     Screenshot 2022-10-17 180326
>   * List name and escape attempts of animals that weigh more than 10.5kg
>     Screenshot 2022-10-17 180408
>

### CODE REVIEW

File: queries.sql
Comment (AbelG101) [inline queries.sql:1]: - [ ] Ouch! this is a close one but the query for the question "list the name of all animals born between 2016 and 2019" should include all the months in 2019 so please update your query from `'2019-01-01'` to `2019/31/12`

File: queries.sql [hunk 1]
```diff
@@ -5,4 +5,4 @@ SELECT * from animals WHERE name IN ('Agumon', 'Pikachu');
 SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
 SELECT * from animals WHERE neutered = true;
```
Comment (AbelG101) [inline queries.sql]: - [ ] Kindly make sure the query for the question "list the name of all animals that are neutered and have less than 3 escape attempts." includes the condition checking if the animal is `neutered` as you're only checking wether the escape attempts is less than 3.

File: queries.sql
Comment (AbelG101) [inline queries.sql:1]: - [ ] Finally, make sure to update your screenshots in the PR description after making the above changes.

Comment (AbelG101) [inline queries.sql:1]: - [ ] Your MIT link is broken so please make sure to fix it as it makes your repo look unprofessional.

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @gilberthappi 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Animal tabe properly created 
- Correct use of Git Flow ✔
- Great PR screenshot summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql
Comment (gilberthappi) [reply queries.sql:1]: Okey Thanks

Comment (gilberthappi) [reply queries.sql:1]: Okey Thanks

File: queries.sql (hunk 1 above)
Comment (gilberthappi) [reply queries.sql]: Okey Thanks

File: queries.sql
Comment (rloterh) [reply queries.sql:1]: well resolved! ✔️

File: queries.sql (hunk 1 above)
Comment (rloterh) [reply queries.sql]: well resolved! ✔️

File: queries.sql
Comment (rloterh) [reply queries.sql:1]: well resolved! ✔️

Comment (rloterh) [reply queries.sql:1]: well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Happi,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 🎯 
🎲 Good job for introducing improvements to the query and ReadMe!

## Optional suggestions 🧰 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

**You can also consider:**
We can make the ReadMe appear more professional by including _Key Features_, _Getting Started_ and _Future Features_ sections to your ReadMe. 

Cheers..Happy coding! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**